src/DataOnev2/DataOnev2.jl

```diff
@@ -0,0 +1,34 @@
+abstract type DataOnev2 <: DataRepo end
+
+export KNB
+include("KNB.jl")
+
+function description(repo::DataOnev2, mainpage)
+    desc = try text_only(matchall(sel"description para", mainpage.root)) catch "" end
+    authors = try [text_only(i) for i in matchall(sel"individualName", mainpage.root) if contains(string(i.parent), "creator")] catch [] end
+    author = format_authors(authors)
+    license = try text_only(first(matchall(sel"intellectualRights", mainpage.root))) catch "" end
+    rawdate = try Dates.Date((text_only(first(matchall(sel"pubDate", mainpage.root)))), "yyyy-mm-dd") catch "" end
+    year = try Dates.year(rawdate) catch "" end
+    date = try Dates.format(rawdate, "U d, yyyy") catch "" end
+    references = ""
+    paper = join(authors, ", ") * " ($(year)) " * data_fullname(repo, mainpage) * " " * references
```

Comment:
This does not give you the name of the paper.
If one can't get the name of the paper, it is fine to exclude it from the generation.
If the DataOne API doesn't keep that kinda metadata then it is not a worry.

Reply:
But this is for the citation. I was already creating this in the previous PR, and as you had termed it is okay, I considered putting it here too.

Reply:
Ah, sorry I messed up in my review there.
I didn't understand.

I think best course is to just skip producing anything about what to cite,
(or maybe just stick the DOI numbers there)

Until we can do something like
I suggest in #22